packages/suite/src/components/suite/NavigationBar/components/NavigationActions/index.tsx

```diff
@@ -134,7 +133,7 @@ const NavigationActions = (props: Props) => {
                                             value: !tor,
```

Comment:
analytics is no more working - probably because of tor<->clearnet change
should be probably moved to action creator or middleware